classes/class-dfrps-cpt.php

```diff
@@ -296,6 +296,16 @@ function add_custom_meta_boxes() {
 			array()
 		);
 
+		add_meta_box(
+			'dfrps_cpt_schedule_metabox',
+			_x( 'Custom Update Schedule', DFRPS_DOMAIN ),
```

Comment:
@growdev  Unfortunately I made the mistake of using the constant `DFRPS_DOMAIN` for the `$domain` argument in translation functions. Unfortunately, I learned later, that this method actually doesn't work. So we should use `'datafeedr-product-sets'` instead of `DFRPS_DOMAIN`. I thought I had fixed that in this plugin but I guess I only fixed this issue the API plugin.  But, it's not a huge deal because we don't ship any translation files anyway. ☹️  Thanks!

Reply:
Ah, good point. I noticed phpcs added this as a warning. I'll add a new issue for this so this PR doesn't get messy.